app/src/main/java/com/example/trempel/network/model/Rating.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.trempel.network.model
```

Comment:
We can have in the same file as `ProductModel` if you like

Reply:
done